skyportal/services/test_server/test_server.py

```diff
@@ -79,6 +80,18 @@ def lt_request_matcher(r1, r2):
     )
 
 
+def lco_request_matcher(r1, r2):
```

Comment:
You'll probably have to update this at some point to look in the body and distinguish between SPECTRAL, MUSCAT, etc, but for now just the URI/method match should be fine since we're just looking at Sinistro.

Reply:
For now we just have Sinistro access, so this should be OK. I am happy to push the YAML... but cache/ is not part of the repo currently.

Reply:
Ah yeah, it can probably go into like `tests/data` and then you can just modify the `get_cache_file()` in the test server code to return that for LCO requests.

Reply:
@kmshin1397 nice. pushed that change. lemme know what you think.

Reply:
@mcoughlin Ah you may want to remove your auth token from the YAML file (I'm assuming that's your real one?). Just replace it with some test string like "testtoken" or whatever, and have that be what the test allocation uses as well. You probably want to clean up the git history for it as well with `bfg` or something (docs on doing that: https://docs.github.com/en/github/authenticating-to-github/removing-sensitive-data-from-a-repository)

Reply:
In this case, rebasing, pausing at the offending commit, editing, and continuing should be enough.  See `git rebase -i`.

Reply:
I replaced with testtoken and also revoked the original one.